### DISCUSSION TITLE
Malay Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#385] [Language Leaders]
@jimmy93 - discussion here about Malay language leaders.

**Comment by zubir2k** [comment]
Hi. We have started working on the files.

**Comment by synesthesiam** [comment]
@anas-ivs, @zubir2k, @wnas86 - continuing discussion about Malay here.

**Comment by synesthesiam** [comment]
@zubir2k are you interested in being a language leader?

**Reply by zubir2k** [reply]
I'd be honored. thank you👌

**Reply by synesthesiam** [reply]
Great! Adding you as a language leader now :slightly_smiling_face:
